test/ziggurat/config_test.clj

```diff
@@ -2,7 +2,7 @@
   (:require [clojure.test :refer :all]
```

Comment:
Add fixtures in this test file, instead of adding (mount/start) and (mount/stop) in every file